File: rust/example1.rs
mod exevalator;
use exevalator::Exevalator;

/// A simple example to use Exevalator.
fn main() {

    // Create an instance of Exevalator Engine
    let mut exevalator = Exevalator::new();

    // Evaluate the value of an expression
    let result: f64 = match exevalator.eval("1.2 + 3.4") {
        Ok(eval_value) => eval_value,
        Err(eval_error) => panic!("{}", eval_error),
    };

    // Display the result
    println!("result: {}", result);    
}

File: typescript/example1.ts
import Exevalator from "./exevalator";

/*
 * A simple example to use Exevalator.
 */

// Create an instance of Exevalator Engine
let exevalator: Exevalator = new Exevalator();

// Evaluate the value of an expression
const result: number = exevalator.eval("1.2 + 3.4");

// Display the result
console.log(`result: ${result}`);
